bitnami/external-dns/templates/deployment.yaml

```
@@ -106,8 +106,10 @@ spec:
         {{- end }}
 
         {{- if eq .Values.provider "google" }}
+        {{- if ne .Values.google.project "" }}
```

Not tested, just a suggestion, would the solution work by grouping both `if` into one?
```suggestion
        {{- if and (eq .Values.provider "google") (ne .Values.google.project "") -}}
```

If yes, can you apply it for everything and remove the unnecessaries final `{{- end }}`?

---

Good suggestion @carrodher, however, if we want to do it like this I suggest we do it consistent across the entire chart since the double if switch seems to be the "preferred" way so far.

---

It depends. If I'm not wrong **Go Templates** evaluates both conditions even if the 1st one is false when using a single `if`.
Therefore, there are cases where it's better to use a "double if"